components/LocationSingle/LocationSingle.js

```diff
@@ -77,7 +128,14 @@ function LocationSingle(props = {}) {
         Component={CampusInfo}
         options={{
           variables: {
-            campusName: campus,
+            campusName:
```

Comment:
@codingwithlis For the campus name, could you add the CFE names to the `campusNameFormatted` util and use that method instead? It would look a bit cleaner